models/reviews.models.js

```diff
@@ -81,3 +81,37 @@ exports.insertComment = (review_id, username, body) => {
       return result.rows[0];
     });
 };
+
+exports.alterVotes = (review_id, inc_votes) => {
+  if (!inc_votes || !/^-?[0-9]*$/.test(inc_votes)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid patch object",
+    });
+  }
```

Comment:
This is absolutely fine - but you would be free to at least try and put something invalid in the database and let it error out - let yourself rely on the error handlers you already have in place